package.json

```diff
@@ -87,6 +91,7 @@
     "mime-db": "^1.52.0",
     "mirror-drive": "^1.3.0",
     "mutexify": "^1.4.0",
+    "path": "npm:bare-node-path",
```

Comment:
remove

Reply:
Same as the other comment. `bare-dev` does not work and throws without this workaround (even with import map).

Reply:
This should now be removed.